### PR TITLE
Adds docker images to execute operators on k8s

### PR DESCRIPTION
**Pull request by saurav-c**
## Describe your changes and why you are making these changes
This PR adds the dockerfiles to create the necessary operator images to run workflows on k8s.

## Related issue number (if any)
ENG 1549

## Checklist before requesting a review
- [ ] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [ ] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

**Review by hsubbaraj-spiral — APPROVED** [review]
Yep looks good to me, did we already push these to the appropriate docker hub repos?

File: src/dockerfiles/connectors/test/snowflake.dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM aqueducthq/base_connector:latest
```
Comment (saurav-c) [inline src/dockerfiles/connectors/test/snowflake.dockerfile:1]: Remove this before merging

File: src/dockerfiles/function/test/function.dockerfile [hunk 2]
```diff
@@ -0,0 +1,34 @@
+FROM continuumio/miniconda3:4.10.3
```
Comment (saurav-c) [inline src/dockerfiles/function/test/function.dockerfile:1]: Remove this before merging

File: src/dockerfiles/Makefile [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+# By default, turn off the Makefile practice of printing each command before
+# you run it.
+ifndef VERBOSE
+.SILENT:
+endif
+
+build-function:
+	docker build . -t aqueducthq/function -f function/function.dockerfile --no-cache
+
+build-function-test:
```
Comment (saurav-c) [inline src/dockerfiles/Makefile:12]: @hsubbaraj-spiral You should be able to run this make command and `make build-snowflake-test` from the `src/dockerfiles` directory

File: src/dockerfiles/connectors/test/start-snowflake.sh [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+git clone https://github.com/aqueducthq/aqueduct.git
```
Comment (saurav-c) [inline src/dockerfiles/connectors/test/start-snowflake.sh:3]: @hsubbaraj-spiral This should cause the code to be fetched every single time the pod is run, so you don't need to repush a new image everytime the code changes.

**Pull request merged**